Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -776,7 +776,7 @@ public void RaiseSourceDetected(IMixedRealityInputSource source, IMixedRealityCo
             HandleEvent(sourceStateEventData, OnSourceDetectedEventHandler);
         }
 
-        private static readonly ExecuteEvents.EventFunction<IMixedRealitySourceStateHandler> OnSourceDetectedEventHandler =
+        public static readonly ExecuteEvents.EventFunction<IMixedRealitySourceStateHandler> OnSourceDetectedEventHandler =
```

Comment:
It would be great if we could get this working without flipping these to `public`. These definitions should really stay private to the input system and not directly referenced outside it.

Reply:
+1

Reply:
With Cameron out at the moment, I'm looking into this

Reply:
Thank you @keveleigh, I just made you a collaborator.

Reply:
Hey @keveleigh, do you have any thoughts on how you would like these handlers encapsulated? I can make the changes needed if you are busy with other tasks.

Reply:
Perhaps we could modify [MixedRealityInputSystem.DispatchEventToObjectFocusedByPointer](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs#L540) to allow for the [event capture phase](https://javascript.info/bubbling-and-capturing#capturing)?

Reply:
I'm not familiar with that method. But, it looks similar to where I'm using handles in: https://github.com/Cameron-Micka/MixedRealityToolkit-Unity/blob/44fb0bd836e18e58024f35181e8c299a8cacfa00/Assets/MixedRealityToolkit/Extensions/EventSystemExtensions.cs#L75